.github/workflows/ci.yml

```diff
@@ -0,0 +1,136 @@
+# Qserv operator CI workflow
+---
+name: "CI"
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    name: Build image
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Declare Variables
+        id: vars
+        shell: bash
+        run: |
+          DIR="$PWD"
+          . ./conf.sh
+          echo "IMAGE=${IMAGE}" >> $GITHUB_ENV
+          echo "IMAGE=${IMAGE}"
+      - name: Build fink-alert-simulator image
+        run: |
+          ./build.sh
+      - name: Export fink-alert-simulator image
+        run: |
+          docker images
+          echo "${{ env.IMAGE }}"
+          mkdir -p artifacts
+          docker save "${{ env.IMAGE }}" > artifacts/image.tar
+          echo "${{ env.IMAGE }}" > artifacts/image-tag
+      - uses: actions/upload-artifact@v2
+        with:
+          name: docker-artifact
+          path: artifacts
+  itest:
+    name: Run integration tests
+    runs-on: ubuntu-22.04
+    needs: build
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Declare Version Variables
+        id: vars
+        shell: bash
+        run: |
+          echo "IMAGE=$(. ./conf.sh && echo $IMAGE)" >> $GITHUB_ENV
+          echo "IMAGE=${IMAGE}"
+          echo "FINK_BROKER_DIR=$(. ./conf.sh && echo $FINK_BROKER_DIR)" >> $GITHUB_ENV
+      - name: Create k8s/kind cluster
+        run: |
+          git clone --depth 1 -b "k8s-1.25.0" --single-branch https://github.com/k8s-school/kind-helper.git
```

Comment:
Is `k8s-1.25.0` a version that will change frequently? If so, probably better to store it in an env variable.

Reply:
This will not change frequently and is only used here.